### DISCUSSION TITLE
A way to run code examples in issues

### DISCUSSION THREAD

**Discussion by DzenanJupic** [community/community#4489] [Projects and Issues]
## The current situation
Often, when an issue is opened, there's some kind of minimal code example that showcases the problem. After the issue is opened, other people then need to recreate the problem locally, which sometimes fails. Maybe since the author has some local software that interferes with the rest, maybe because the author forgot to mention a few steps, or maybe because the code example itself doesn't lead to the bug.

## Possible solution
Wouldn't it be great if issues would offer some kind of GitHub Actions like interface? Basically, a way for the author to recreate the problem in a sealed environment, like a Docker container.

### Advantages
1. The author can (or has to) run the code beforehand, and instantly knows if the code example leads to the bug
2. Other contributors can just have a look at the output from the container and don't necessarily have to recreate the problem locally
3. If other contributors want the recreate the problem locally, they have exact instructions (i.e. from a DockerFile) that are known to actually recreate the issue

#### What are your thoughts on this?

**Comment by My69goodtimes** [comment]
Wtf
